### DISCUSSION TITLE
How do I improve search speed of FLAT index?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#26105] [Q&A and General discussion]
How do I improve search speed of FLAT index? Is it possible?
Or is the only choice is to deploy a cluster?

**Comment by yhmo** [comment]
Assigning more CPU cores to the host, and increasing the segment size can slightly improve the search performance of FLAT.
With cluster mode, you can deploy more query nodes to compute parallelly.
